superset/db_engine_specs/oracle.py

```diff
@@ -58,3 +63,18 @@ def epoch_to_dttm(cls) -> str:
     @classmethod
     def epoch_ms_to_dttm(cls) -> str:
         return "TO_DATE('1970-01-01','YYYY-MM-DD')+(1/24/60/60/1000)*{col}"
+    
+    @classmethod
+    def fetch_data(cls, cursor: Any, limit: int) -> List[Tuple]:
+        """
+        :param cursor: Cursor instance
+        :param limit: Maximum number of rows to be returned by the cursor
+        :return: Result of query
+        """
+        if not cursor.description:
+            return []
+        if cls.arraysize:
+            cursor.arraysize = cls.arraysize
+        if cls.limit_method == LimitMethod.FETCH_MANY:
+            return cursor.fetchmany(limit)
+        return cursor.fetchall()
```

Comment:
To avoid deuplicating code, do you think this work?
```suggestion
        super().fetch_data(cursor, limit)
```
Also, extra points for adding a unit test! 😉